script/setup/runhcs-version

```diff
@@ -1 +1 @@
-v0.12.0
+1d406d0eac5573287ba7b46a04a58275410137ac
```

Comment:
Is it possible to get a release with the changes? 2.0 doesn't seem too far off so it'd be nice to stay on a tag

Reply:
Hmm, I wasn't planning to get this in containerd 2.0 since the rc builds are already out. That's why we didn't get a release on hcsshim. Do we still need a tag if we don't want to get this in 2.0?

Reply:
Maybe I've forgotten the release process but I'd think if this got checked in anytime before we decide to tag 2.0 then it'd be part of the release

Reply:
Yeah, it does look like merging this right now will get it into 2.0. I have confirmed that getting this into 2.0 shouldn't be a problem.

Reply:
Gotcha. 

> I have confirmed that getting this into 2.0 shouldn't be a problem.

You mean the commit instead of a tag?

Reply:
We don't need to get this into 2.0, we can merge this after the final 2.0 tag is cut. At that time, we can merge with a commit ID of hcsshim and then when containerd is preparing for a 2.1 release we can make a separate hcsshim tag at that time and vendor that. Does that sound okay?

Reply:
Sounds fine to me at least, I'm not sure if any other maintainers would prefer us stay on a tag.

Reply:
I agree it would be best to keep a tagged release here.